project/app/views.py

```diff
@@ -19,6 +19,46 @@ def has_permission(self, request, view):
             print(e)
 
 
+class RequiredManager(BasePermission):
+    def has_permission(self, request, view):
+        user = request.user.role
+        try:
+            if str(user) == 'M':
+                return Response({})
+        except Exception as e:
+            print(e)
+
+
+class RequiredEmployee(BasePermission):
+    def has_permission(self, request, view):
+        role = request.POST.get("role")
+        try:
+            if str(role) == 'E':
+                return Response({})
+        except Exception as e:
+            print(e)
+
+
+class AdminCanCreate(BasePermission):
+    def has_permission(self, request, view):
+        role = request.POST.get("role")
```

Comment:
Put this inside try block

Reply:
is it fine to use two try blocks?

Reply:
We can but not needed, Just move the line inside existing try block